app/_src/production/cp-deployment/systemd.md

```diff
@@ -36,38 +36,38 @@ ExecStart = /path/to/kuma/bin/kuma-cp run --config-file=/home/kuma/cp-config.yam
 #
 # for Kuma demo setup, we chose the same limit as `docker` and `containerd` set by default.
 # See https://github.com/containerd/containerd/issues/3201
-LimitNOFILE=1048576
-Restart=always
-RestartSec=1s
+LimitNOFILE = 1048576
+Restart = always
+RestartSec = 1s
 # disable rate limiting on start attempts
-StartLimitIntervalSec=0
-StartLimitBurst=0
+StartLimitIntervalSec = 0
+StartLimitBurst = 0
 
 [Install]
 WantedBy = multi-user.target
 ```
 
 ## kuma-dp
 
-```
+```systemd
 [Unit]
-Description=Kuma data plane proxy
-After=network.target
-Documentation=https://kuma.io
+Description = {{ site.mesh_product_name }} Data Plane Proxy
+After = network.target
+Documentation = {{ site.links.web }}
 
 [Service]
-User=kuma
-ExecStart=/home/kuma/kong-mesh-1.9.1/bin/kuma-dp run \
+User = {{ site.mesh_product_name_lowercase }}
+ExecStart = /home/{{ site.mesh_product_name_lowercase }}/{{ site.mesh_product_name_lowercase }}-{{ page.latest_version }}/bin/kuma-dp run \
```

Comment:
`page.latest_version` won't work for dev / current HEAD but I guess this is fine, people can replace that.

Reply:
I agree, this is just example, you still need to provide your dp template as well as the token.